static/js/modules/filter-panel.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var FilterSet = require('./filter-set').FilterSet;
+var accessibility = require('fec-style/js/accessibility');
+
+var defaultOptions = {
+  body: '.filters',
+  form: '#category-filters',
+  toggle: '#filter-toggle'
+};
+
+function FilterPanel(options) {
+  this.isOpen = false;
+  this.options = _.extend({}, defaultOptions, options);
+
+  this.$body = $(this.options.body);
+  this.$form = $(this.options.form);
+  this.$toggle = $(this.options.toggle);
+
+  this.$toggle.on('click', this.toggle.bind(this));
+
+  this.filterSet = new FilterSet(this.$form).activate();
+  if (!_.isEmpty(this.filterSet.serialize())) {
+    this.show();
+  }
+
+  this.adjust();
+}
+
+FilterPanel.prototype.adjust = function() {
+  if ($('body').width() > 768) {
+    this.show();
+  } else if (!this.isOpen) {
+    this.hide();
+  }
+};
+
+FilterPanel.prototype.show = function() {
+  this.$body.addClass('is-open');
+  this.$toggle.addClass('is-active');
+  this.$toggle.find('.filters__toggle__text').html('Hide filters');
+  accessibility.restoreTabindex(this.$form);
+  $('body').addClass('is-showing-filters');
+  this.isOpen = true;
+};
+
+FilterPanel.prototype.hide = function() {
+  this.$body.removeClass('is-open');
+  this.$toggle.removeClass('is-active');
+  this.$toggle.find('.filters__toggle__text').html('Show filters');
+  $('#results tr:first-child').focus();
```

Comment:
I could see this selector being a potential problem in that it's accessing global information (a #results selector) and is using an ID, so you can only have one on a page.

That being said, the filter panel doesn't sound like a component that would be included twice and "results" is pretty straightforward so I don't think this is a required fix.